src/summaries/level-two-routes.component.tsx

```diff
@@ -56,6 +57,11 @@ export const levelTwoRoutes: PatientChartRoute[] = [
     url: "/patient/:patientUuid/chart/conditions",
     component: ConditionsBriefSummary,
     name: "Conditions"
+  },
+  {
+    url: "/patient/:patientUuid/chart/conditions/:conditionUuid",
+    component: ConditionsDetailedSummary,
+    name: "Condition"
```

Comment:
I think this should be "Conditions"

Reply:
This route is for the detailed view when you click on an individual condition - so the singular form should be appropriate for it.

Reply:
In "Chart -> Conditions -> Renal Failure", the "Conditions" refers to the detailedSummary widget, so the link is to "Conditions". The choice of name within a breadcrumb is independent of the final node in the breadcrumb